DOCS/man/options.rst

```diff
@@ -1605,6 +1610,20 @@ Video
         - ``--video-aspect-override=16:9`` or ``--video-aspect-override=1.7777``
         - ``--no-video-aspect-override`` or ``--video-aspect-override=no``
 
+    .. note::
+
+        Since mpv 0.40, a builtin lua script is now loaded by default to provide
+        compatibility for setting the option to ``0`` or ``-1`` by using
+        ``--video-aspect-method`` for these values and printing deprecation
+        warnings. This can be disabled with ``--load-aspect-compat=no``.
```

Comment:
Maybe we want to disable it by default and allow enabling for compatibility?

Also, I didn't follow the discussion closely, but this feels to me more appropriate to the C code and not a script (but if everyone wants a script then sure).

Or maybe just don't depracate thsese values for this option?